docs/developing.md

```diff
@@ -244,14 +242,22 @@ By running the above-mentioned command, it will automatically set up all the pre
 
 #### Post Setup Requirements
 
-After you have done setting up pre-commit checks, then you may need to fulfill a few more requirements to finish setting up the development environment with `NixOS dev-shell`. These include:
+Inside `flake.nix`, the `$HOME/.cargo/bin` is added to the `PATH`. The git hook manager for this project is `rusty-hook`, and (unfortunately) it has to be installed via `cargo` and can't be neatly packaged up inside the flake ([see issue](https://github.com/swellaby/rusty-hook/issues/168)). This was automatically installed when you ran `cargo test`.
 
-- `Cargo-watch` installed on your system which will allow you to auto-build the project when any checks occur in the source code files in the codebase (`websurfx` directory). Before you install `cargo-watch` on your system, make sure you have `cargo` installed on your system. To install `cargo-watch` run the following command:
+Run `git commit`. This should take a while and the pre-commit hook will get rejected and we will explain why. 
```

Comment:
I think I would suggest reframing this part a little because it seems to be repeated two times, which is not good. Also, as mentioned in the first code review, I think I would suggest making it a little straight forward, something like this. :slightly_smiling_face: :

``` 
Finally, you will need to install the following development dependencies to get the pre-commit fully setup....
```